### DISCUSSION TITLE
Getting "Page not found" when accessing Keycloak Account Console

### DISCUSSION THREAD

**Discussion by pnalepa-bishopfox** [keycloak/keycloak#17043] [Q&A]
I want to set up the Keycloak instance locally on MacOS Ventura. I'm following this guide: https://www.keycloak.org/getting-started/getting-started-docker and almost everything is working well except for I cannot log in to the KeyCloak Account Console as a user. I'm able to log in and use the KeyCloak Admin console without any issues. I'm not sure, what's missing.

### The URL I'm using for logging in to the Keycloak Account Console
```
http://localhost:8080/realms/myrealm/protocol/openid-connect/auth?client_id=myclient&redirect_uri=https%3A%2F%2Fwww.keycloak.org%2Fapp%2F%23url%3Dhttp%3A%2F%2Flocalhost%3A8080%26realm%3Dmyrealm%26client%3Dmyclient&state=9a72957e-c6c3-4bbf-918b-a6cae974ea31&response_mode=fragment&response_type=code&scope=openid&nonce=3f94f46e-9fbf-4069-9330-07bcdbddadef
```

![Screenshot 2023-02-13 at 15 57 43](https://user-images.githubusercontent.com/122298692/218492380-6ba04e87-6a67-4a36-8244-3b9a4525db49.png)

Any help or suggestions would be higly appreciated.

**Comment by DAHAG-ArisNourbakhsh** [comment]
The url you posted is not for the account-console. 

That url is a sign-in attempt into the client you need to create first in the [Secure Your First App](https://www.keycloak.org/getting-started/getting-started-docker#_secure_your_first_app) Step. What happened is that your local keycloak instance reacted with a 404 to the 'http://localhost:8080/realms/myrealm' portion of the request as you very most likely forgot to create that realm as outlined in the guide at the [beginning](https://www.keycloak.org/getting-started/getting-started-docker#_create_a_realm).

**Reply by pnalepa-bishopfox** [reply]
Ok, the things I've done are:
1. I used the latest version of KeyCloak image (newer one than the one I spotted an issue in)
2. It started working for me

**Comment by LMonert** [comment]
To get to the account console of a realm you use a URI of the following format: /realms/**nameOfRealm**/account
Keycloak will then redirect you to a login page with the correct URL query parameters.
